multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+/*.yaml
+*~
```

Comment:
NM. Just saw your previous comment that it's emacs temp files. That's fine, but still wonder about the *.yaml